src/assets/scss/components/_typography.scss

```diff
@@ -105,8 +105,8 @@ article {
     }
 }
 
-// Accented link with arrow
-a.link--accent {
+// Accented link with arrow right
+a.link--accent-right {
```

Comment:
Since these are being used for directional arrows now, let's just be obvious and name them as such across the app. Something like `link-arrow--right` and `link-arrow--left`.